fcrepo-import-export-driver/src/main/java/org/fcrepo/importexport/ArgParser.java

```diff
@@ -126,10 +129,23 @@ Config parseConfiguration(final String[] args) {
         return config;
     }
 
-    @Override
-    public Exporter parse(final String[] args) {
+    /**
+     * Parse command-line arguments.
+     * @return A configured Importer or Exporter instance.
+    **/
+    public TransferProcess parse(final String[] args) {
+
+        final Config config = parseConfiguration(args);
+        if (config.isImport()) {
+            return new Importer(config, clientBuilder());
+        } else if (config.isExport()) {
+            return new Exporter(config, clientBuilder());
+        }
+        return null;
```

Comment:
Instead of returning `null`, please throw an exception... maybe an `IllegalArgumentException`.